docs/detections/how-to-query-alert-indices.asciidoc

```diff
@@ -0,0 +1,60 @@
+[[how-to-query-alert-indices]]
+[role="xpack"]
+== How to query alert indices
+This page explains how you should query alert indices, for example when building rule queries, custom dashboards, or visualizations. (For more information about alert event field definitions, review the [Alert schema].)
+
+The naming conventions for alert indices and their aliases differ depending on which version of {stack} you're running (<8.0 vs 8.0+), but there are aliases that you can query regardless of which version you're using:
```

Comment:
It seems more consistent to use `<` and `>=` (instead of `<` and `+`), but I wonder if we could omit the operators to make it sound less mathy, more human-readable.
```suggestion
The naming conventions for alert indices and their aliases differ depending on which version of {stack} you're running (before 8.0 or 8.0 and later), but there are aliases that you can query regardless of which version you're using:
```